src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -105,6 +105,7 @@ typedef struct uct_ib_mlx5_bf {
         void                    *ptr;
         uintptr_t               addr;
     } reg;
+    unsigned                    size;       /* BlueFlame register size */
```

Comment:
but it's not actually used as a size anywhere, right? we use the compile-time constant UCT_IB_MLX5_BF_REG_SIZE
so it's used only to enable/disable bf